### PR TITLE
Update CollectiblesList component

### PR DESCRIPTION
**Pull request by graszka22**
This component is updated for this screen: https://zpl.io/b6LQPjg. And by the way this one https://zpl.io/VOePwQX (it's the same component).

### CODE REVIEW

File: src/components/CollectiblesList/CollectiblesList.js [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+`;
+
+const Name = styled(BaseText)`
+  ${fontStyles.small};
+  letter-spacing: ${fontTrackings.small};
+  width: 100%; 
+  text-align: center;
+`;
+
+const Card = styled.TouchableOpacity`
```
Comment (vyvil) [inline src/components/CollectiblesList/CollectiblesList.js:77]: So no more shadows on cards?
(I remember there was an issue with Zeplin, and those were not included in designs, yet required. So just checking if this is not such case)

File: src/components/CollectiblesList/CollectiblesList.js [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+const EmptyStateWrapper = styled.View`
+  align-items: center;
+  justify-content: center;
+  padding: 20px;
+  flex-grow: 1;
+`;
+
+const Name = styled(BaseText)`
+  ${fontStyles.small};
+  letter-spacing: ${fontTrackings.small};
```
Comment (vyvil) [inline src/components/CollectiblesList/CollectiblesList.js:72]: Guessing, this is a legacy code - in current designs there are no letter spacing :)

File: src/components/CollectiblesList/CollectiblesList.js [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+const CollectiblesList = ({
+  collectibles, onCollectiblePress, isSearching, theme, activeAccountAddress,
+}: Props) => {
+  const [isReceiveModalVisible, setIsReceiveModalVisible] = useState<boolean>(false);
+
+  const renderCollectible = ({ item }: CollectibleItem) => {
+    const { name, image } = item;
+
+    const icon = image;
```
Comment (vyvil) [inline src/components/CollectiblesList/CollectiblesList.js:103]:
```suggestion
    const { name, image: icon } = item;
```

File: src/screens/Assets/WalletView.js [hunk 4]
```diff
@@ -297,10 +302,10 @@ class WalletView extends React.Component<Props, State> {
           {activeTab === COLLECTIBLES && (
             <CollectiblesList
               collectibles={this.getFilteredCollectibles()}
-              searchQuery={query}
-              navigation={navigation}
-              activeAccountAddress={activeAccountAddress}
-            />)}
+              onCollectiblePress={this.handleCollectiblePress}
+              isSearching={!!query}
```
Comment (vyvil) [inline src/screens/Assets/WalletView.js:306]: Now it isn't changing much, but I guess, technically it should be:
```suggestion
              isSearching={query.length >= MIN_QUERY_LENGTH}
```
(as only then collectibles are being filtered)

**Review by vyvil — APPROVED** [review]
☀️
Just some notes:

File: src/components/CollectiblesList/CollectiblesList.js (hunk 1 above)
Comment (graszka22) [reply src/components/CollectiblesList/CollectiblesList.js:76]: we can remove them https://zpl.io/b6LQPjg

**Pull request merged**